[2023-09-19 07:53:15,998] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:37:00+00:00 [queued]>
[2023-09-19 07:53:16,107] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:37:00+00:00 [queued]>
[2023-09-19 07:53:16,120] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:53:16,120] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2023-09-19 07:53:16,120] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-09-19 07:53:16,222] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2023-09-19T01:37:00+00:00
[2023-09-19 07:53:16,317] {standard_task_runner.py:52} INFO - Started process 1470 to run task
[2023-09-19 07:53:16,447] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', '2023-09-19T01:37:00+00:00', '--job-id', '245', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_pipeline.py', '--cfg-path', '/tmp/tmp2k6o_jgd', '--error-file', '/tmp/tmpruhd03xy']
[2023-09-19 07:53:16,449] {standard_task_runner.py:77} INFO - Job 245: Subtask fetch_weather_data
[2023-09-19 07:53:16,998] {logging_mixin.py:104} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data 2023-09-19T01:37:00+00:00 [running]> on host 92beb7708765
[2023-09-19 07:53:17,922] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data_pipeline
AIRFLOW_CTX_TASK_ID=fetch_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-19T01:37:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-19T01:37:00+00:00
[2023-09-19 07:53:19,653] {python.py:151} INFO - Done. Returned value was: True
[2023-09-19 07:53:19,998] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, execution_date=20230919T013700, start_date=20230919T075316, end_date=20230919T075319
[2023-09-19 07:53:20,708] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2023-09-19 07:53:20,925] {local_task_job.py:151} INFO - Task exited with return code 0
